src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1176,6 +1178,10 @@ protected String checkJavaVersion(final PrintStream logger, String javaCommand,
         return null;
     }
 
+    private double requiredJavaVersion() {
+        return Double.valueOf("1." + JDK_MAJOR_VERSION);
```

Comment:
as jenkins.io offers a jdk list - i recommend using this list to choose a JDK version